package.json

```diff
@@ -1,4 +1,21 @@
 {
+  "name": "interceptor",
+  "version": "0.2.1",
+  "description":
+    "A browser extension that mocks AJAX request at the browser level so you can run front ends without really starting a backend server.",
+  "license": "MIT",
+  "homepage": "https://github.com/code-mancers/interceptor#readme",
+  "repository": {
```

Comment:
I repository is already there and no need for change

Reply:
i just moved the same thing up. You can notice the deleted lines below  😄

Reply:
in url `.git` is missing. 
Ideally this should be the url to `git` repo not its code home page.